src/pages/ErrorBoundry.tsx

```diff
@@ -0,0 +1,39 @@
+import { AddressZero } from '@ethersproject/constants';
+
+import { NimiPage, NimiPageProvider, ToastProvider } from '@nimi.io/card';
+import { Loader, LoaderWrapper } from 'components/Loader';
+import { OpacityMotion } from 'components/motion';
+import { AnimatePresence } from 'framer-motion';
+import { useInitialtNimiData } from 'hooks/useDefaultNimiData';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+export const ErrorBoundary = () => {
+  const { nimiUsername: ensName } = useParams();
+
+  // Fetch the Nimi data for the ENS name
+  const { data: initialNimi, isGenerated } = useInitialtNimiData({
+    ensName: ensName!,
+    account: AddressZero,
```

Comment:
At some point, remove this parameter. The ENS name should be sufficient to resolve the address

Reply:
Also this is just for route error do you know how I can potentially catch this one or remove it?
![Screenshot 2023-04-10 at 00 05 15](https://user-images.githubusercontent.com/33226956/230798710-674d4e08-529c-4d1c-9b3f-bd85cc82026a.png)